src/Modules/Microsoft.Azure.Functions.PowerShellWorker/Microsoft.Azure.Functions.PowerShellWorker.psm1

```diff
@@ -141,3 +141,91 @@ function New-OrchestrationCheckStatusResponse {
         }
     }
 }
+
+<#
+.SYNOPSIS
+    Send an external event to an orchestration instance.
+.DESCRIPTION
+    Send an external event with the given event name, and event data to an orchestration instance with the given instance ID.
+.EXAMPLE
+    PS > Send-DurableExternalEvent -InstanceId "example-instance-id" -EventName "ExampleExternalEvent" -EventData "data for the external event"
+    Return the instance id of the new orchestration.
+.PARAMETER InstanceId
+    The ID of the orchestration instance that will handle the external event.
+.PARAMETER EventName
+    The name of the external event.
+.PARAMETER EventData
+    The JSON-serializable data associated with the external event.
+.PARAMETER TaskHubName
+    The TaskHubName of the orchestration instance that will handle the external event.
+.PARAMETER ConnectionName
+    The name of the connection string associated with TaskHubName
+#>
+function Send-DurableExternalEvent {
+    [CmdletBinding()]
+    param(
+        [Parameter(
+            Mandatory=$true,
+            Position=0,
+            ValueFromPipelineByPropertyName=$true)]
+        [ValidateNotNullOrEmpty()]
+        [string] $InstanceId,
+
+        [Parameter(
+            Mandatory=$true,
+            Position=1,
+            ValueFromPipelineByPropertyName=$true)]
+        [string] $EventName,
+
+        [Parameter(
+            Position=2,
+            ValueFromPipelineByPropertyName=$true)]
+        [object] $EventData,
+
+		[Parameter(
+            Position=3,
```

Comment:
It is usually a good idea to be careful with positional parameters, assign positions to the most obvious and intuitive parameters only, and limit the number of positional parameters to 1-3. Otherwise, this encourages writing code passing a bunch of unnamed parameters, which becomes error-prone and hard to read. In this specific case, I recommend removing positions from TaskHubName and ConnectionName, so the user will have to specify parameter names explicitly.